spec/features/dashboard/advisers_edit_spec.rb

```diff
@@ -0,0 +1,113 @@
+RSpec.feature 'The dashboard firm edit page' do
+  include CheckboxGroupHelpers
+
+  let(:advisers_index_page) { Dashboard::AdvisersIndexPage.new }
+  let(:adviser_edit_page) { Dashboard::AdviserEditPage.new }
+  let(:original_postcode) { 'L1 2NH' }
+  let(:updated_postcode) { 'EH11 2AB' }
+
+  before do
+    5.times { |num| Accreditation.create!(name: Faker::Name.name, order: num) }
+    5.times { |num| Qualification.create!(name: Faker::Name.name, order: num) }
+    5.times { |num| ProfessionalStanding.create!(name: Faker::Name.name, order: num) }
+    5.times { |num| ProfessionalBody.create!(name: Faker::Name.name, order: num) }
```

Comment:
Yip, didn't realise there was existing factories. Need to be better at looking in mas-rad_core. Thanks @munckymagik